firedrake/interpolation.py

```diff
@@ -137,7 +218,7 @@ def _interpolator(V, dat, expr, subset, access):
             raise NotImplementedError("Interpolation onto another mesh not supported.")
         if expr.ufl_shape != V.shape:
             raise ValueError("UFL expression has incorrect shape for interpolation.")
-        ast, oriented, needs_cell_sizes, coefficients, _ = compile_ufl_kernel(expr, to_pts, coords, coffee=False)
+        ast, oriented, needs_cell_sizes, coefficients, _ = compile_ufl_kernel(expr, to_pts, V.ufl_element(), coords, coffee=False)
```

Comment:
Based on what @miklos1 pointed out in https://github.com/firedrakeproject/tsfc/pull/192, the additional argument `V.ufl_element()` can be removed? Since this is getting ignored.